src/sentry/integrations/github/integration.py

```diff
@@ -146,6 +147,18 @@
 ERR_INTEGRATION_PENDING_DELETION = _(
     "It seems that your Sentry organization has an installation pending deletion. Please wait ~15min for the uninstall to complete and try again."
 )
+ERR_INTEGRATION_INVALID_INSTALLATION = _(
+    "Your GitHub account does not have owner privileges for the chosen organization."
+)
+ERR_MISSING_INTEGRATION = _(
+    "The chosen GitHub installation could not be found. Please uninstall the Sentry app from your GitHub account settings."
```

Comment:
Thoughts on wording?

Reply:
can we tell them to also install from the sentry side?